src/resources/TeamContributionCalendar/TeamContributionCalendar.test.js

```diff
@@ -109,6 +118,26 @@ describe('TeamContributionCalendar', () => {
         calendarHeader,
       )).to.equal(true);
     });
+
+    it('gets the tooltip element', () => {
```

Comment:
This doesn't test anything useful.

Reply:
I don't really get it.

Could you explain your statement above, please?

Reply:
Whether `GetStyledCalendarElement.tooltip` was called is not testing the behavior.

Reply:
So it's unnecessary to test function calls whenever there isn't any particular parameter?

Reply:
Even with parameters, the use case for testing a function call is that if the call has side effects and those cannot be verified directly. There's no point in testing a getter this way. What should be tested is that the tooltips are there.

Reply:
Cool, thanks for the clarification!